Sources/Utilities/ViewControllerFactory.swift

```diff
@@ -0,0 +1,15 @@
+import Authentication
+import GDSCommon
+import UIKit
+
+final class ViewControllerFactory {
+    func createIntroViewController(session: LoginSession) -> IntroViewController {
+        let viewModel = OneLoginIntroViewModel {
+            let url = URL.oneLoginAuthorize
+            let configuration: LoginSessionConfiguration = LoginSessionConfiguration.oneLogin(url: url)
```

Comment:
Looking at this now we can probably remove the explicit type for the `configuration` property as it will infer it from the return type for `LoginSessionConfiguration.oneLogin(url:)`